launch/startExploring.launch

```diff
@@ -3,6 +3,10 @@
     <node pkg="ma_rrt_path_plan" type="main.py" name="ma_rrt_path_plan_node" output="screen">
         <param name="desiredWaypointsFrequency" value="5" type="int" />
 
+        <!-- TODO: Comment me out -->
+        <!-- <param name="odom_topic" value="/odometry/real"/>
+        <param name="world_frame" value="map"/> -->
```

Comment:
These three lines are confusing. 
1. For who is this TODO then if it is there
2. Generally having the frames specified in launch files is cool, but the frames that you specified are unique to your system (again). For this purposes you could create a separate launch file, but I dont see a reason to have it this repo.